test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -928,6 +911,50 @@ public void ApplyApplicationInsightsConfig_Sets_MaxTelemetryItemsPerSecond()
             Assert.Equal(25, scriptConfig.ApplicationInsightsSamplingSettings.MaxTelemetryItemsPerSecond);
         }
 
+        [Fact]
+        public void ApplyHostHealthMonitorConfig_AppliesExpectedSettings()
+        {
+            JObject config = JObject.Parse("{ }");
+
+            ScriptHostConfiguration scriptConfig = new ScriptHostConfiguration();
+            ScriptHost.ApplyConfiguration(config, scriptConfig);
+
+            Assert.True(scriptConfig.HostHealthMonitor.Enabled);
+            Assert.Equal(TimeSpan.FromSeconds(15), scriptConfig.HostHealthMonitor.HealthCheckInterval);
+            Assert.Equal(TimeSpan.FromMinutes(15), scriptConfig.HostHealthMonitor.HealthCheckWindow);
+            Assert.Equal(30, scriptConfig.HostHealthMonitor.HealthCheckThreshold);
+
+            // now set custom configuration and verify
+            config = JObject.Parse(@"
+            {
+                'healthMonitor': {
+                    'enabled': false
+                }   
+            }");
+            scriptConfig = new ScriptHostConfiguration();
+            ScriptHost.ApplyConfiguration(config, scriptConfig);
+
+            Assert.False(scriptConfig.HostHealthMonitor.Enabled);
+
+            config = JObject.Parse(@"
+            {
+                'healthMonitor': {
+                    'enabled': true,
+                    'healthCheckInterval': '00:00:07',
+                    'healthCheckWindow': '00:07:00',
+                    'healthCheckThreshold': 77
+                }
+                
+            }");
+            scriptConfig = new ScriptHostConfiguration();
+            ScriptHost.ApplyConfiguration(config, scriptConfig);
+
+            Assert.True(scriptConfig.HostHealthMonitor.Enabled);
+            Assert.Equal(TimeSpan.FromSeconds(7), scriptConfig.HostHealthMonitor.HealthCheckInterval);
+            Assert.Equal(TimeSpan.FromMinutes(7), scriptConfig.HostHealthMonitor.HealthCheckWindow);
+            Assert.Equal(77, scriptConfig.HostHealthMonitor.HealthCheckThreshold);
+        }
+
```

Comment:
Perhaps just a personal nit but why isn't this 3 tests? We're checking 3 different behaviors...